TestKit/Sources/TestKit/XCTestCase+Wait.swift

```diff
@@ -34,9 +36,9 @@ extension XCTestCase {
     /// }
     /// ```
     ///
-    func waitUntil(file: StaticString = #file,
+    public func waitUntil(file: StaticString = #file,
                    line: UInt = #line,
-                   timeout: TimeInterval = Constants.expectationTimeout,
+                   timeout: TimeInterval = 5.0,
                    condition: @escaping (() -> Bool)) {
```

Comment:
nit: similar to above, these 4 lines could be further indented from `public` 🙂

Reply:
Thanks! Fixed in 6281cb611ec64e9ed77652769964b901136c4841.